src/index.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
```

Comment:
Please make use of extensions suggested by Harish. Code formatter, code spell checker, Prettier etc.